src/lib.rs

```diff
@@ -1,4 +1,7 @@
 // Lib is present to allow for benchmarking
+#[macro_use]
+extern crate log;
```

Comment:
As this is edition 2018, you can instead use `log::info!("Hello")` (or import `use log::info`) everywhere and not worry about `#[macro_use]` here. Personally I find I like seeing `log::error!("danger")` instead of `error!("danger")` anyway.

Reply:
Thanks for the heads up @mehcode! 😄